### ISSUE TITLE
build fails with ghc-9.6

### ISSUE THREAD

**Issue by juhp** [xtendo-org/rawfilepath#8]
```
       Building library for rawfilepath-1.0.1..                                                                         [591/167012]
       [ 1 of 11] Compiling Data.ByteString.RawFilePath                                                                             
                                                                                                                                    
       /var/stackage/work/unpack-dir/unpacked/rawfilepath-1.0.1-40eb4a7f7ec779d24d080df399725ab25b45ee668acd744bf0283f8b01a2c117/src
/Data/ByteString/RawFilePath.hs:52:21: error: [GHC-83865]                                                                           
           • Couldn't match expected type: OpenFileFlags -> IO Fd                                                                   
                         with actual type: IO Fd                                                                                    
           • The function ‘openFd’ is applied to four value arguments,                                                              
               but its type ‘RawFilePath -> OpenMode -> OpenFileFlags -> IO Fd’                                                     
               has only three                                                                                                       
             In the expression: openFd path ReadOnly Nothing defaultFlags                                                           
             In a case alternative:                                                                                                 
                 ReadMode -> openFd path ReadOnly Nothing defaultFlags
          |
       52 |         ReadMode -> openFd path ReadOnly Nothing defaultFlags
          |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        
       /var/stackage/work/unpack-dir/unpacked/rawfilepath-1.0.1-40eb4a7f7ec779d24d080df399725ab25b45ee668acd744bf0283f8b01a2c117/src
/Data/ByteString/RawFilePath.hs:52:42: error: [GHC-83865]
           • Couldn't match expected type ‘OpenFileFlags’
                         with actual type ‘Maybe a0’
           • In the third argument of ‘openFd’, namely ‘Nothing’
             In the expression: openFd path ReadOnly Nothing defaultFlags
             In a case alternative:
                 ReadMode -> openFd path ReadOnly Nothing defaultFlags
          |
       52 |         ReadMode -> openFd path ReadOnly Nothing defaultFlags
          |                                          ^^^^^^^
        
       /var/stackage/work/unpack-dir/unpacked/rawfilepath-1.0.1-40eb4a7f7ec779d24d080df399725ab25b45ee668acd744bf0283f8b01a2c117/src
/Data/ByteString/RawFilePath.hs:54:23: error: [GHC-83865]
           • Couldn't match expected type: OpenFileFlags -> IO Fd
                         with actual type: IO Fd
           • The function ‘openFd’ is applied to four value arguments,
               but its type ‘RawFilePath -> OpenMode -> OpenFileFlags -> IO Fd’
               has only three
             In the expression:
               openFd path WriteOnly (Just stdFileMode) appendFlags
             In a case alternative:
                 AppendMode -> openFd path WriteOnly (Just stdFileMode) appendFlags
          |
       54 |         AppendMode -> openFd path WriteOnly (Just stdFileMode) appendFlags
:
:
```

**Comment by juhp** [comment]
See #7

**Comment by xtendo-org** [comment]
Thanks. This is fixed in the latest `main`.

Let me close this issue after a release.

**Comment by xtendo-org** [comment]
Fixed in 1.1.0

Stackage re-enable pull request: https://github.com/commercialhaskell/stackage/pull/7102
